utils_general/aa-countries.R

```diff
@@ -0,0 +1,75 @@
+###
```

Comment:
Not regarding this line, but the directory structure. I would propose to not place this in utils_general as this are really pipeline scripts. Instead I would propose to make a `analyses/glb` folder, and place it there. What do you think?

Reply:
Oh I hadn't noticed glb. Much better location for it.

Reply:
I need to figure out how to move the Rproj file with it..

Reply:
`glb` doesn't exist yet😁 but think this is a perfect occasion to create it